src/tests/ftest/util/test_utils_pool.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, context, log=None, cb_handler=None, dmg_command=None):
         # used.
         self.control_method = BasicParameter(self.USE_API, self.USE_API)
         uname = getpass.getuser()
-        gname = grp.getgrnam(uname)[0]
+        gname = grp.getgrgid(self.gid)[0]
```

Comment:
This approach was removed in the recently merged https://github.com/daos-stack/daos/pull/2073.  I suspect this PR is no longer needed.  Can you confirm the issue is not seen with the latest master?